pkg/apis/bookkeeper/v1alpha1/bookkeepercluster_types.go

```diff
@@ -702,6 +710,10 @@ func (bk *BookkeeperCluster) BookkeeperTargetImage() (string, error) {
 	return fmt.Sprintf("%s:%s", bk.Spec.Image.Repository, bk.Status.TargetVersion), nil
 }
 
+func (bk *BookkeeperCluster) HeadlessServiceNameForBookie() string {
```

Comment:
can we not have this method within the util itself?

Reply:
if we keep method in util, we have to pass extra param , where as in bookkeeper_cluster_types.go, we have the bk cluster and can get the value easily